linkerd/failure-accrual/src/test/scala/io/buoyant/linkerd/failureAccrual/ConsecutiveFailuresTest.scala

```diff
@@ -2,10 +2,27 @@ package io.buoyant.linkerd.failureAccrual
 
 import com.twitter.finagle.liveness.FailureAccrualPolicy
 import com.twitter.finagle.util.LoadService
-import io.buoyant.linkerd.FailureAccrualInitializer
+import com.twitter.conversions.time._
+import io.buoyant.config.Parser
+import io.buoyant.linkerd.{FailureAccrualConfig, FailureAccrualInitializer}
 import io.buoyant.test.FunSuite
+import org.scalacheck.Gen
+import org.scalatest.Matchers
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+
+class ConsecutiveFailuresTest extends FunSuite
+  with Matchers
```

Comment:
There's a couple places in this file where I'm actually still using `Matchers` for testing the structure of collections. I think they're more expressive than assertions & that code would get a bit more complex with just `assert`